{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/urls.py

```diff
@@ -1,8 +1,8 @@
 from django.urls import path
 
-from {{ cookiecutter.project_slug }}.users.views import user_detail_view
-from {{ cookiecutter.project_slug }}.users.views import user_redirect_view
-from {{ cookiecutter.project_slug }}.users.views import user_update_view
+from .views import user_detail_view
+from .views import user_redirect_view
+from .views import user_update_view
```

Comment:
If we do keep the `force-single-line` rule for imports, we might want to change this style of importing views, and instead do:
```python
from . import views

urlpatterns = [
    path("~redirect/", view=views.user_redirect_view, name="redirect"),
    ...
```

This is one case where I think the issue of import blocks being too long could be an actual problem #4880